std/stdio.d

```diff
@@ -2466,12 +2466,6 @@ $(D Range) that locks the file and allows fast writing to it.
         _iobuf* handle_;     // the unshared version of fps
         int orientation_;
     public:
-        deprecated("accessing fps/handle/orientation directly can break LockingTextWriter integrity")
```

Comment:
The policy is about 1 year as deprecated and documented, and 1 year as deprecated and undocumented. This was undocumented back in 2013 even before it was deprecated. Yes, we would ideally put a note in the documentation about when it's going to go away, but even then, the date that we normally give for removal is actually the date that it's removed from the docs, not from the code, and if something has been deprecated for ~1.5 years and undocumented for longer than that, then no one should be using it by this point anyway. They've been given more than ample warning.